tracee-ebpf/main.go

```diff
@@ -798,23 +798,55 @@ func prepareEventsToTrace(eventFilter *tracee.StringFilter, setFilter *tracee.St
 		}
 	}
 	for _, name := range excludeEvents {
-		id, ok := eventsNameToID[name]
-		if !ok {
-			return nil, fmt.Errorf("invalid event to exclude: %s", name)
+		// Handle event prefixes with wildcards
+		if strings.HasSuffix(name, "*") {
+			found := false
+			prefix := name[:strings.Index(name, "*")]
+			for event, id := range eventsNameToID {
+				if strings.HasPrefix(event, prefix) {
+					isExcluded[id] = true
+					found = true
+				}
+			}
+			if !found {
+				return nil, fmt.Errorf("invalid event to exclude: %s", name)
+			}
+		} else {
+			id, ok := eventsNameToID[name]
+			if !ok {
+				return nil, fmt.Errorf("invalid event to exclude: %s", name)
+			}
+			isExcluded[id] = true
 		}
-		isExcluded[id] = true
 	}
 	if len(eventsToTrace) == 0 && len(setsToTrace) == 0 {
 		setsToTrace = append(setsToTrace, "default")
 	}
 
 	res = make([]int32, 0, len(tracee.EventsIDToEvent))
 	for _, name := range eventsToTrace {
-		id, ok := eventsNameToID[name]
-		if !ok {
-			return nil, fmt.Errorf("invalid event to trace: %s", name)
+		// Handle event prefixes with wildcards
+		if strings.HasSuffix(name, "*") {
+			var ids []int32
+			found := false
+			prefix := name[:strings.Index(name, "*")]
```

Comment:
Since `*` is the suffix, you can replace `strings.Index(name, "*")` with `len(name)-1`

Reply:
- [ ] replace strings.Index(name, "*") with len(name)-1